docs/auto-qc/iqms_intepretability.md

```diff
@@ -0,0 +1,1288 @@
+      <td>1</td>
+      <td>1</td>
+      <td>-1.165106</td>
+      <td>1.337207</td>
+      <td>0.707725</td>
+      <td>-0.946513</td>
+      <td>0.806795</td>
+      <td>0.998398</td>
+      <td>0.969088</td>
+      <td>0.277350</td>
+      <td>...</td>
+      <td>-0.844802</td>
+      <td>-0.259740</td>
+      <td>0.359484</td>
+      <td>0.940510</td>
+      <td>-0.914859</td>
+      <td>-0.933136</td>
+      <td>1.314992</td>
+      <td>0.870827</td>
+      <td>-0.036182</td>
+      <td>-0.299338</td>
+    </tr>
+    <tr>
+      <th>sub-992238_acq-headmotion1_T1w</th>
+      <td>3</td>
+      <td>2</td>
+      <td>-0.336329</td>
+      <td>0.212102</td>
+      <td>0.022202</td>
+      <td>0.914001</td>
+      <td>0.424312</td>
+      <td>0.288854</td>
+      <td>0.540691</td>
+      <td>0.374432</td>
+      <td>...</td>
+      <td>0.177875</td>
+      <td>0.370348</td>
+      <td>0.767916</td>
+      <td>-0.232326</td>
+      <td>0.213313</td>
+      <td>0.214843</td>
+      <td>0.473483</td>
+      <td>0.148033</td>
+      <td>0.496048</td>
+      <td>1.543252</td>
+    </tr>
+    <tr>
+      <th>sub-992238_acq-headmotion2_T1w</th>
+      <td>3</td>
+      <td>3</td>
+      <td>0.744436</td>
+      <td>-0.927664</td>
+      <td>0.535875</td>
+      <td>0.505978</td>
+      <td>-0.087667</td>
+      <td>-0.188586</td>
+      <td>-0.088070</td>
+      <td>0.038501</td>
+      <td>...</td>
+      <td>1.002042</td>
+      <td>-0.535891</td>
+      <td>0.727136</td>
+      <td>-1.210021</td>
+      <td>1.171428</td>
+      <td>1.189201</td>
+      <td>0.475541</td>
+      <td>-0.590150</td>
+      <td>-0.422853</td>
+      <td>0.869156</td>
+    </tr>
+    <tr>
+      <th>sub-992238_acq-standard_T1w</th>
+      <td>1</td>
+      <td>1</td>
+      <td>-1.131714</td>
+      <td>1.398085</td>
+      <td>-0.721208</td>
+      <td>-0.946513</td>
+      <td>0.174032</td>
+      <td>0.137035</td>
+      <td>0.358339</td>
+      <td>-0.047950</td>
+      <td>...</td>
+      <td>-1.000083</td>
+      <td>-0.697419</td>
+      <td>0.331397</td>
+      <td>0.594432</td>
+      <td>-0.796867</td>
+      <td>-0.740462</td>
+      <td>0.560316</td>
+      <td>0.929091</td>
+      <td>0.559520</td>
+      <td>2.280824</td>
+    </tr>
+  </tbody>
+</table>
+<p>436 rows × 64 columns</p>
+</div>
+
+
+
+### Visualizing the data (always!)
+
+Rule number one of data analysis: always visualize your data! We can plot the pairwise scatterplots to get an idea of the relationships between the IQMs. There are quite a few IQMs, so plottings this might take a few minutes, depending on your hardware.
+
+
+```python
+metrics_sets = {f"set{i+1}": list(iqms_scaled.columns[i*10:i*10+9]) for i in range(6)}
+metrics_sets["set8"] = list(iqms_scaled.columns[60:])
+for metrics_set in metrics_sets.values():
+     sns.pairplot(iqms_scaled[metrics_set])
+```
+
+
+    
+![png](copy_interpretability-of_iqms_files/copy_interpretability-of_iqms_24_0.png)
```

Comment:
I'm not sure you need this, the notebook will be run on the data.

Reply:
Let's see when we push the notebook.